google-cloud-storage/src/main/java/com/google/cloud/storage/multipartupload/model/UploadPartRequest.java

```diff
@@ -85,6 +88,18 @@ public String uploadId() {
     return uploadId;
   }
 
+  /**
+   * Returns the CRC32C checksum of the part to upload.
```

Comment:
Can you add the encoding related information. It should be clear to the user what encoding it needs to do on the crc32c value.